### ISSUE TITLE
Won't build against libav on Debian Wheezy

### ISSUE THREAD

**Issue by acgourley** [tmarrinan/node-demux#8]
This is mostly just to help people who run into a problem where npm install will complain that AVPixelFormat isn't defined. It's because Debian Wheezy uses libav which doesn't use the 'AV' prefix before it's enum constants. 

I solved the issue by moving my containers to debian jessie which uses ffmpeg instead of libav to provide these headers.

We can likely solve this with compile-time checks on the version of libavutil package present on the system. Possibly #defines can solve it as well.

**Comment by tmarrinan** [comment]
Yes - it was designed for using FFmpeg libraries. Even on Linux systems that use libav, FFmpeg can be installed (usually requires adding an extra repo).
